apps/web/components/eventtype/EventSetupTab.tsx

```diff
@@ -262,12 +262,14 @@ export const EventSetupTab = (
                   if (!newOptions.find((opt) => opt.value === defaultDuration?.value)) {
                     if (newOptions.length > 0) {
                       setDefaultDuration(newOptions[0]);
+                      formMethods.setValue("length", newOptions[0].value);
```

Comment:
When setting default duration, UI was showing a value but form was not impacted. Fixed.